packages/modules/itwin-viewer-react/src/components/app-ui/frontstages/DefaultFrontstage.tsx

```diff
@@ -192,6 +192,7 @@ export class DefaultFrontstage extends FrontstageProvider {
         }
         rightPanel={<StagePanel allowedZones={[6, 9]} />}
         bottomPanel={<StagePanel allowedZones={[7, 8, 9]} />}
+        leftPanel={<StagePanel allowedZones={[1, 4, 7]} />}
```

Comment:
idk if 1 should be an allowed zone, typically used for tools, but ill defer to Kevin

Reply:
the panel "should show up to the left of the tools if all are being utilized